test/fixtures/us_core_examples/Patient-ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7.json

```diff
@@ -0,0 +1,139 @@
+{
+    "id":"ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7",
+    "meta":{
+       "profile":[
+          "http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient"
+       ]
+    },
+    "text":{
+    },
+    "extension":[],
+    "identifier":[
+       {
+          "system":"https://github.com/synthetichealth/synthea",
+          "value":"ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7"
+       },
+       {
+          "type":{
+             "coding":[
+                {
+                   "system":"http://terminology.hl7.org/CodeSystem/v2-0203",
+                   "code":"MR",
+                   "display":"Medical Record Number"
+                }
+             ],
+             "text":"Medical Record Number"
+          },
+          "system":"http://hospital.smarthealthit.org",
+          "value":"ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7"
+       },
+       {
+          "type":{
+             "coding":[
+                {
+                   "system":"http://terminology.hl7.org/CodeSystem/v2-0203",
+                   "code":"SS",
+                   "display":"Social Security Number"
+                }
+             ],
+             "text":"Social Security Number"
+          },
+          "system":"http://hl7.org/fhir/sid/us-ssn",
+          "value":"999-53-4901"
+       },
+       {
+          "type":{
+             "coding":[
+                {
+                   "system":"http://terminology.hl7.org/CodeSystem/v2-0203",
+                   "code":"DL",
+                   "display":"Driver's License"
+                }
+             ],
+             "text":"Driver's License"
+          },
+          "system":"urn:oid:2.16.840.1.113883.4.3.25",
+          "value":"S99943125"
+       }
+    ],
+    "name":[
+       {
+          "use":"official",
+          "family":"Becker968",
+          "given":[
+             "Sadie426"
+          ],
+          "prefix":[
+             "Mrs."
+          ]
+       },
+       {
+          "use":"maiden",
+          "family":"Harber290",
+          "given":[
+             "Sadie426"
+          ],
+          "prefix":[
+             "Mrs."
+          ]
+       }
+    ],
+    "telecom":[
+       {
+       }
+    ],
```

Comment:
Are you expecting this to be an error or warning?

Reply:
This was a mistake on my part, readded.